resources/views/accordion.blade.php

```diff
@@ -8,12 +8,15 @@
     'circleClass' => '',
     'circleSize' => 'sm',
     'toggleTitle' => false,
+    'mainWrapperClass' => null,
+    'descriptionListClass' => null,
+    'slotWrapperClass' => null,
 ])
 
 @php ($isDark = $dark)
 
-<div x-data="{ openPanel: null }">
-    <dl>
+<div x-data="{ openPanel: null }" class="{{ $mainWrapperClass }}">
+    <dl class="{{ $descriptionListClass }}">
```

Comment:
might be better to check if it's set, otherwise it will output an empty class property. E.g.

```
<dl class="">
```

![image](https://user-images.githubusercontent.com/8069294/123071192-596ddb80-d40c-11eb-9586-76065c5b8c45.png)